### DISCUSSION TITLE
Env variable issues?

### DISCUSSION THREAD

**Discussion by moleculezz** [bluenviron/mediamtx#2599] [Questions]
### Question

Hi,

I've setup mediamtx on Digital Ocean App Platform which uses docker.
I'm trying to setup some ENV variables, but it seems as if the paths ENV variables are not working perhaps.

I used the following syntax to setup a path:
`MTX_PATHS_STUDIO_PUBLISHUSER`
`MTX_PATHS_STUDIO_PUBLISHPASS`

But trying to publish a stream to `/studio` with the username and password isn't working.

I tried `MTX_RTSPADDRESS` and this did work when I checked.

So, am I doing something wrong with the paths variable?

**Comment by aler9** [comment]
Hello, please attach a server log with `logLevel: debug`, as explicitly mentioned in the issue instructions, otherwise it's not possible to understand where the problem is.

[a comment by an automated account is not shown]
